src/message.rs

```diff
@@ -31,7 +30,7 @@ fn msg_send_fn<R: Any>() -> unsafe extern fn(*mut Object, Sel, ...) -> R {
     }
 }
 
-#[cfg(target_arch = "x86")]
+#[cfg(all(target_arch = "x86", not(feature = "gnustep_runtime")))]
 fn msg_send_super_fn<R: Any>() -> unsafe extern fn(*mut Object, Sel, ...) -> R {
```

Comment:
The GNUstep runtime doesn't implement sending messages to super via `objc_msgSendSuper`. The GNUstep-specific implementation for sending messages to super is at ll. 182–194.